backend-api/src/functions/asyncActiveSession/tokenService/tests/tokenService.test.ts

```diff
@@ -22,6 +23,7 @@ describe("Token Service", () => {
   beforeEach(() => {
     dependencies = {
       publicKeyGetter: () => new MockPubicKeyGetterGetPublicKeySuccess(),
+      tokenVerifier: () => new MockTokenVerifierVerifySuccess(),
```

Comment:
Do we need a test block for when tokenVerifier.verify returns an error result?

Reply:
Thanks @jmooney-dd. I've added this test